apps/core/src/utils/transaction/getTransactionAction.ts

```diff
@@ -4,27 +4,39 @@
 
 import { IotaTransactionBlockResponse } from '@iota/iota-sdk/client';
 import { TransactionAction } from '../../interfaces';
-import { checkIfIsTimelockedStaking } from '../stake/checkIfIsTimelockedStaking';
+import { checkIfIsTimelockedStaking } from '../stake';
+import { isMigrationTransaction, isSupplyIncreaseVestingCollectTransaction } from '..';
 
 export const getTransactionAction = (
     transaction: IotaTransactionBlockResponse,
     currentAddress?: string,
 ) => {
+    const sender = transaction.transaction?.data.sender;
     const {
         isTimelockedStaking,
         isTimelockedUnstaking,
         stakeTypeTransaction,
         unstakeTypeTransaction,
     } = checkIfIsTimelockedStaking(transaction?.events);
 
-    if (stakeTypeTransaction) {
+    const { isMigration } = isMigrationTransaction(transaction.transaction);
+    const { isSupplyIncreaseVestingCollect } = isSupplyIncreaseVestingCollectTransaction(
```

Comment:
This is not necessary related to supply increase, this is related to the timelock package, so I would not call it like so. Lets better call is `isUnlockTimelockedObjectTransaction` or so